Source/WebKit/WPEPlatform/wpe/WPEView.h

```diff
@@ -119,6 +120,12 @@ WPE_API void                    wpe_view_resized                       (WPEView
 WPE_API gdouble                 wpe_view_get_scale                     (WPEView      *view);
 WPE_API void                    wpe_view_scale_changed                 (WPEView      *view,
                                                                         gdouble       scale);
+WPE_API gboolean                wpe_view_get_visible                   (WPEView      *view);
+WPE_API void                    wpe_view_set_visible                   (WPEView      *view,
+                                                                        gboolean      visible);
+WPE_API gboolean                wpe_view_get_mapped                    (WPEView      *view);
```

Comment:
I think the names are too generic, and thus confusing. Reading only `wpe_view_get_mapped` - I would have no idea what it does...

Reply:
Since I cannot propose anything better _and_ it's existing X11/Gtk terminology, let's stick with it.